sections/06_discussion.md

```diff
@@ -6,9 +6,28 @@ can be further pruned.*
+Small fractions of the genome are implicated in biochemical activities such as transcription factor (TF) binding and histone modifications. For example, less than 1% of the genome can be confidently labeled as bound for most transcription factors. Therefore, it is important to evaluate the genome-wide recall and false discovery rate (FDR) of classification models of biochemical activities. Targeted validation experiments of specific biochemical activities usually necessitate an FDR of 5%-25%. When predicted biochemical activities are used as features in other models, such as gene expression models, a low FDR may not be as critical if the downstream models can satisfy their evaluation criteria; an FDR of 50% in this context may suffice.
+
+What is the correspondence between these metrics and commonly used classification metrics such as auPRC and auROC? auPRC evaluates the average precision, or equivalently, the average FDR across all recall thresholds. This metric provides an overall estimate of performance across all possible use cases which can be misleading for targeted validation experiments. For example, classification of TF binding sites can exhibit a recall of 0% at 10% FDR and auPRC greater than 0.6. In this case, the auPRC may be competitive but the predictions are prohibitive for targeted validation that can only examine a couple of predictions. auROC evaluates the average recall across all false positive rate (FPR) thresholds which is often a highly misleading metric in this domain. For example, consider a classification model with recall of 0% at FDR less than 25% and 100% recall at FDR greater than 25%. In the context of TF binding predictions where only 1% of genomic regions are bound by the TF, this is equivalent to a recall of 100% for FPR greater than 0.33%. In other words, the auROC would be 0.9967 and not indicative of the predictive utility for targeted validation. It is not unusual to obtain a chromosome-wide auROC greater than 0.99 for TF binding predictions but a recall of 0% at 10% FDR. 
+
+#### Formulation of classification labels
+Genome-wide continuous signals are commonly formulated into classification labels through signal peak detection. For example, peaks in ChIP-seq signal are used to identify locations of TF binding and histone modifications. Such procedures rely on thresholding criteria to define what constitutes a peak in the signal. This inevitably results in a set of signal peaks that are close to the threshold, not sufficient to constitute a positive label, and too similar to positively labeled examples to constitute a negative label. To avoid an arbitrary label for these example they may be labeled as “ambiguous”. Ambiguously labeled examples can then be ignored during model training and evaluation of recall and FDR. The correlation between model predictions on these examples and their signal values can be used to evaluate if the model correctly ranks these examples between positive and negative examples.
```

Comment:
@jacklanchantin @bdo311 Do you think we should remove this from the TF binding section since similar discussion came up there?  We could refer to this section instead.